configstack/running_module.go

```diff
@@ -154,6 +168,8 @@ func runModules(modules map[string]*runningModule) error {
 		waitGroup.Add(1)
 		go func(module *runningModule) {
 			defer waitGroup.Done()
+			module.Module.TerragruntOptions.ErrWriter = io.MultiWriter(&module.OutStream, &module.ErrStream)
+			module.Module.TerragruntOptions.Writer = &module.OutStream
```

Comment:
I'm a bit confused by this... You're overwriting `TerragruntOptions.ErrWriter` to write to both `module.OutStream` and `module.ErrStream`... But what was the `Terragrunt.LOptions.ErrWriter` value set to before that? Are `module.OutStream` and `module.ErrStream` initialized to anything? Will this buffer those errors until the very end or stream to `stdout` / `stderr`?

Same questions go for `TerragruntOptions.Writer`, with the additional one of what happens when you point a second item to `module.OutStream`?